pkg/etl/config/etl_profile.h

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright (C) 2022 Jens Wetterich <jens@wetterich-net.de>
+*
+* This file is subject to the terms and conditions of the GNU Lesser
+* General Public License v2.1. See the file LICENSE in the top level
+* directory for more details.
+*/
+
+/**
+* @ingroup     pkg_etl
+* @brief
+* @{
+*
+* @brief       Common settings for etl
+* @details     This activates the usage of the STL if available and/or the usage of workarounds for missing NAN macros.
+*              This is configured by compiler defines the in Makefile.include
+* @author      Jens Wetterich <jens@wetterich-net.de>
+*/
+#if !defined(ETL_PROFILE_H) && !defined(DOXYGEN)
```

Comment:
```suggestion
#ifndef ETL_PROFILE_H
```

Just use the standard header guard, doxygen will be fine.

Reply:
The intention was to not need to document the defines. They are only for internal use in the ETL and not to be used anywhere in RIOT... 🤔

Reply:
Changed it accordingly and documented the defines as `@internal`.